### PR TITLE
Releasing version 12.0.0.Dev03

### PR DESCRIPTION
**Pull request by karesti**

### CODE REVIEW

**Comment by karesti** [comment]
@ryanemerson @pruivo change whatever needs to be changed here please and when the artifacts will be present in maven central, I will push the website

File: _config/ispn.yml [hunk 1]
```diff
@@ -50,16 +50,16 @@ stable:
...
-  maven_latest: 12.0.0.Dev01
-  release_date: 2020/07/27
-  fixed_issues: https://issues.redhat.com/secure/ReleaseNote.jspa?projectId=12310799&version=12346555
+  version: 12.0.0.Dev03
+  maven_latest: 12.0.0.Dev03
+  release_date: 2020/09/8
+  fixed_issues: https://issues.redhat.com/secure/ReleaseNote.jspa?projectId=12310799&version=12347011
   codename: N/A
-  whats_new: Hibernate Search 6 + Lucene 8, console improvements, etc...
+  whats_new: Documentation improvement, Cross Site Replication Conflict Resolutionn, Console improvements
```
Comment (pruivo) [inline _config/ispn.yml:58]: I think this is "what's new in ISPN 12" and not in the release.

**Comment by pruivo** [comment]
there is `12.0.0.Dev01` in the file that needs to be updated.

File: _config/ispn.yml (hunk 1 above)
Comment (karesti) [reply _config/ispn.yml]: reverted

**Comment by ryanemerson** [comment]
Thanks @karesti

**Pull request merged**